Shopify/cli
# Port fetchThemeAssets / fetchChecksums to graphQL

### WHY are these changes introduced?

Migrate off the REST Assets API and use graphQL instead.

### WHAT is this pull request doing?

This PR adjusts the `fetchThemeAsset` and `fetchChecksums` method to use the [`OnlineStoreTheme.files` field](https://shopify.dev/docs/api/admin-graphql/unstable/objects/OnlineStoreTheme#connection-files).

Where appropriate I've also updated callers to fetch multiple assets at once with `fetchThemeAssets`.

### How to test your changes?

`p shopify theme pull ...` should fetch all the files from your theme to a local directory.

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
